cassandra/metadata.py

```
@@ -2610,12 +2624,61 @@ def __init__(self, connection, timeout, fetch_size, metadata_request_timeout):
+        if self._scylla_consistency_cache is not None:
+            return self._scylla_consistency_cache
+
+        if self._is_not_scylla():
+            self._scylla_consistency_cache = {}
+            return self._scylla_consistency_cache
+
+        consistency_by_ks = {}
+        try:
+            rows = self._query_build_rows(self._SELECT_SCYLLA_KEYSPACES, lambda row: row)
+            consistency_by_ks = {row["keyspace_name"]: row.get("consistency") for row in rows}
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)